storage-node/src/services/webApi/app.ts

```diff
@@ -186,3 +202,27 @@ function verifyUploadTokenData(accountAddress: string, token: UploadToken, reque
     throw new Error('Nonce not found')
   }
 }
+
+async function validateUploadFileParams(req: express.Request, res: express.Response<unknown, AppConfig>) {
+  const { api, queryNodeEndpoint, workerId } = res.locals
+
+  const storageBucketId = Number(req.query.storageBucketId)
+  const dataObjectId = Number(req.query.dataObjectId)
+  const bagId = req.query.bagId!.toString()
+
+  const parsedBagId = parseBagId(bagId)
+
+  const [dataObject] = await Promise.all([
+    api.query.storage.dataObjectsById(parsedBagId, Number(dataObjectId)),
```

Comment:
`dataObjectId` is already a `Number`:
```suggestion
    api.query.storage.dataObjectsById(parsedBagId, dataObjectId),
```

Reply:
addressed in [aa7d7c6](https://github.com/Joystream/joystream/pull/4402/commits/aa7d7c66903226558bf95492578d3ac64eff62e6)